static/sass/_pattern_strip.scss

```diff
@@ -984,5 +984,8 @@
     * {
       z-index: 2;
     }
+    > div a {
```

Comment:
Can you make this a little more specific? Add a class to the link?

Reply:
done